python/cuml/dask/common/dask_arr_utils.py

```diff
@@ -29,14 +29,21 @@
 from dask.distributed import default_client
 from toolz import first
 
-from cuml.dask.common.input_utils import _extract_partitions
+from cuml.dask.common.part_utils import _extract_partitions
 
 from cuml.utils import rmm_cupy_ary
 
 from dask.distributed import wait
 from dask import delayed
 
 
+def validate_dask_array(darray, client=None):
```

Comment:
Add a small comment here about the purpose of this function (that it's intended to be reusable by algorithms that accept dask array as input).